service/server.py

```diff
@@ -26,6 +76,17 @@ def get_title(title_ref):
         #Title not found
         abort(404)
 
+@app.route('/title_search_postcode/<postcode>', methods=['GET'])
+def get_properties(postcode):
+    postcode.replace ("_", " ")
```

Comment:
Are we sure this is the sanitising we need to do here? :) I am expecting a lot more sanitising going on potentially in the digital-register-frontend, then passing a postcode that we know is in an acceptable format here?